docs/roman/tweakreg/README.rst

```diff
@@ -0,0 +1,293 @@
+to generate the astrometric reference catalog for all the measured sources
+in the combined field-of-view of the set of input images.
+The default catalog is the latest Gaia DR but `tweakreg` also supports a
+subset of the astronomical catalogs stored at the Barbara A. Mikulski Archive for
+Space Telescopes (MAST). The catalogs currently available through this interface are:
+
+- the Hubble Source Catalog (HSC);
+
+- the GALEX Catalog (V2 and V3);
+
+- the Gaia (DR1, DR2, and DR3) and TGAS Catalogs;
+
+- the TESS Input Catalog (TIC);
+
+- the TESS Candidate Target List (CTL);
+
+- the Disk Detective Catalog;
+
+- the PanSTARRS Catalog (DR1 and DR2);
+
+- the All-Sky PLATO Input Catalog (DR1).
+
+For more information about the MAST Catalogs, please visit `MAST CATALOGS`_.
+
+.. _MAST CATALOGS: https://mast.stsci.edu
+
+The combined source catalog derived in the first step
+then gets cross-matched and fit to this astrometric reference catalog.
+The pipeline initially supports fitting to any Gaia DR (1, 2, or 3).
+The results of this fit then gets back-propagated to all the
+input images to align them all to the astrometric reference frame while
+maintaining the relative alignment between the images.
+
+For this part of alignment, instead of using the default option (Gaia), users can
+supply their own external reference catalog by providing a path to an existing
+file. User-supplied catalog must contain two columns named ``'RA'`` and ``'DEC'`` which
+indicate reference source world coordinates (in degrees). An optional column
+in the catalog is the ``'weight'`` column, which when present, will be used
+in fitting. The catalog must be in a format automatically recognized by
+:py:meth:`~astropy.table.Table.read`.
+
+WCS Correction
+--------------
+The linear coordinate transformation computed in the previous step
+is used to define tangent-plane corrections that need to be applied
+to the GWCS pipeline in order to correct input image WCS.
+This correction is implemented by inserting a ``v2v3corr`` frame with
+tangent plane corrections into the GWCS pipeline of the image's WCS.
+
+Step Arguments
+--------------
+The ``tweakreg`` step has the following optional arguments:
+
+**Catalog parameters:**
+
+* ``save_catalogs``: A boolean indicating whether or not the catalogs should
+  be written out. This parameter is ignored for input data models whose
+  ``meta.tweakreg_catalog`` is a non-empty string pointing to a user-supplied
+  source catalog. (Default=`False`)
+
+* ``use_custom_catalogs``: A boolean that indicates whether
+  to ignore source catalog in the input data model's ``meta.tweakreg_catalog``
+  attribute. If `False`, new catalogs will be generated by the ``tweakreg``
+  step. (Default=`False`)
+
+* ``catalog_format``: A `str` indicating catalog output file format.
+  (Default='ecsv')
+
+* ``catfile``: Name of the file with a list of custom user-provided catalogs.
+  (Default='')
+
+**Optimize alignment order:**
```

Comment:
Is this variable something we use right now?  I think we've only discussed L2 alignment to Gaia; I don't know what this means in this context.

Reply:
I updated the docs to remove the option for a user-provided catalog.

Reply:
I agree that I'm focused on the "optimize alignment order" bit here, which doesn't have meaning to me at L2.  I think we're currently imagining not needing further tweaking at L3, but we'll see.  I wasn't really advocating for removing anything, but I wanted the docs to address our usual case, where the alignment order isn't obviously a thing.